crherman7/rechunk
# feat: admin dashboard

## Scope



[closes TICKET-11](https://github.com/crherman7/rechunk/issues/11)

## Implementation

Utilize [remix.run](https://remix.run/) to bootstrap our admin dashboard. Currently implemented:

- landing page
- dashboard layout
- resource endpoints scaffolding
- database models
- deployment



## Screenshots

<details>
  <summary>ios</summary>

  

![Simulator Screenshot - iPhone 15 - 2024-09-29 at 11 48 43](https://github.com/user-attachments/assets/0ce59639-6bf9-4958-95f1-d756fc5fddae)
</details>

<details>
  <summary>android</summary>
  
  

![Screenshot_1727625072](https://github.com/user-attachments/assets/6622b728-69d4-477e-a3cf-7e0810760868)
</details>

<details>
  <summary>web</summary>

  

![Screen Shot 2024-09-29 at 11 55 42](https://github.com/user-attachments/assets/442b896d-f9d9-4b16-9446-fedfbdf46861)
![Screen Shot 2024-09-29 at 11 55 38](https://github.com/user-attachments/assets/b6ec9ee1-7035-4b90-9f7e-a8a9bd8c56ee)
</details>

## How to Test

Run the example app to ensure nothing is broken dependency-wise since this a monorepo. Run the admin site from the `apps/www` directory.



## Emoji Guide

**For reviewers: Emojis can be added to comments to call out blocking versus non-blocking feedback.**

E.g: Praise, minor suggestions, or clarifying questions that don’t block merging the PR.

> 🟢 Nice refactor!

> 🟡 Why was the default value removed?

E.g: Blocking feedback must be addressed before merging.

> 🔴 This change will break something important

| | | |
| --- | --- | --- |
| Blocking | 🔴 ❌ 🚨 | RED |
| Non-blocking | 🟡 💡 🤔 💭 | Yellow, thinking, etc |
| Praise | 🟢 💚 😍 👍 🙌 | Green, hearts, positive emojis, etc |
